src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/in/msg/ProcessedEnvelope.java

```diff
@@ -9,8 +9,17 @@
 public class ProcessedEnvelope {
 
     public final UUID id;
+    public final String processedCcdReference;
+    public final String processedCcdType;
 
-    public ProcessedEnvelope(@JsonProperty("id") UUID id) {
+    public ProcessedEnvelope(
+        @JsonProperty("id") UUID id,
+        @JsonProperty("processed_ccd_reference") String processedCcdReference,
+        @JsonProperty("processed_ccd_type") String processedCcdType
```

Comment:
will the orchestrator ignore these new fields ?

Reply:
other way around, process will ignore